tasks/pipeline.py

```diff
@@ -9,7 +9,17 @@
 
 @task
 def trigger(ctx, git_ref="master", release_version_6="nightly", release_version_7="nightly-a7", repo_branch="nightly"):
-    pipeline_id = trigger_agent_pipeline(git_ref, release_version_6, release_version_7, repo_branch)
+    pipeline_id = trigger_agent_pipeline(git_ref, release_version_6, release_version_7, repo_branch, deploy=True)
+    wait_for_pipeline("DataDog/datadog-agent", pipeline_id)
+
+
+@task
+def run_with_all_tests(
+    ctx, git_ref="master", here=False, release_version_6="nightly", release_version_7="nightly-a7", repo_branch="none"
+):
```

Comment:
I think we should add a docstring here, very briefly describing the task and showing an example on how to use it. We should also add it on the `trigger` one.